pkg/controller/che/che_controller.go

```diff
@@ -499,18 +505,24 @@ func (r *ReconcileChe) Reconcile(request reconcile.Request) (reconcile.Result, e
 			return *result, err
 		}
 	}
+	if pluginRegistryURL != instance.Status.PluginRegistryURL {
+		instance.Status.PluginRegistryURL = pluginRegistryURL
+		if err := r.UpdateCheCRStatus(instance, "status: Plugin Registry URL", pluginRegistryURL); err != nil {
```

Comment:
Ah, I see the block deleted from line 491 has moved down here. So this is the fallback value if the guess doesn't work?

Reply:
more precisely, if an external URL is provided we use it (even if the embedded registry is deployed => a guessed value is available, but finally not used).